### ISSUE TITLE
Error while parsing archive contents

### ISSUE THREAD

**Issue by Wikinaut** [borgbase/vorta#215]
[UPDATE: A file browser appears to be implemented by the the "Extract" button, but this throws an Error as described further below]

I would like to suggest to add a tiny file browser to navigate in a (temporary FUSE mounted) backup. In addition it would be nice to enable a file "copy" for a file out of the selected backup.

In addition to the "Mount" button there could be a directory/file navigation button.

![grafik](https://user-images.githubusercontent.com/1151915/53687966-49fa8e00-3d3c-11e9-8457-6841ac759555.png)

**Comment by m3nu** [comment]
We already have 2 ways to achieve that:

1. Mount the whole repo or one archive and use your normal file browser
2. Choose *Extract* (Extrahieren) and you will also see a list of folders and files which can be restored.

I would recommend option 1 for restoring a few files and option 2, if you want a full restore with metadata.

<img width="840" alt="screen shot 2019-03-03 at 09 47 05" src="https://user-images.githubusercontent.com/3916435/53689831-7ee8fe00-3d99-11e9-8509-d78754592657.png">

**Comment by Wikinaut** [comment]
@m2nu Uh I overlooked this button, because it was not apparent for what it could be used.

Now I clicked on "Extrahieren" (Extract). A message is then shown 

* "Extracting..." ("Hole Archiv-Inhalt") (very briefly)
* "Done" ("Archiv holen erledigt.")

Very very briefly I could see a line "ERROR....".

but no directory tree was shown. I analysed the situation and the reason was, that my NAS path ```/mnt/nas/``` was apparently not mounted at that moment.

After mounting, I can click onto "Extract", the response changes in that it now takes a while, a message is shown for some seconds ("Hole Archiv-Inhalt"), but no file explorer.

What's wrong here?

**Comment by Wikinaut** [comment]
@m3nu In a console window I saw

```
raceback (most recent call last):
  File "/home/benutzer/.local/lib/python3.6/site-packages/vorta/views/archive_tab.py", line 322, in list_archive_result
    window = ExtractDialog(result['data'], archive)
  File "/home/benutzer/.local/lib/python3.6/site-packages/vorta/views/extract_dialog.py", line 39, in __init__
    files_with_attributes = [parse_line(l) for l in fs_data.split('\n')[:-1]]
  File "/home/benutzer/.local/lib/python3.6/site-packages/vorta/views/extract_dialog.py", line 39, in <listcomp>
    files_with_attributes = [parse_line(l) for l in fs_data.split('\n')[:-1]]
  File "/home/benutzer/.local/lib/python3.6/site-packages/vorta/views/extract_dialog.py", line 28, in parse_line
    size, modified, full_path = line.split('\t')
ValueError: not enough values to unpack (expected 3, got 1)

```

**Comment by m3nu** [comment]
After reading the archive files from Borg, they are parsed to build the tree view. In your case a line fails to parse for some reason. (Could be an error message?)

I did a fix to ignore such cases in #216 . If Borg can read your archive content, it should work now. If there is some other issue and Borg can't access your archive for some reason (not mounted, etc), you will find another error in the log:

- Linux: `$HOME/.cache/Vorta/log`

**Comment by Wikinaut** [comment]
I conform that https://github.com/borgbase/vorta/pull/216 fixes the bug.

**Comment by Wikinaut** [comment]
> After reading the archive files from Borg, they are parsed to build the tree view. In your case a line fails to parse for some reason. (Could be an error message?)
> 
> I did a fix to ignore such cases in #216 . If Borg can read your archive content, it should work now. If there is some other issue and Borg can't access your archive for some reason (not mounted, etc), you will find another error in the log:
> 
>     * Linux: `$HOME/.cache/Vorta/log`
correction:

~/.cache/Vorta/log/vorta.log
